packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuItem.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { IconProps } from 'react-magma-icons';
+import { AccordionItem, AccordionItemProps } from '../Accordion';
+export interface DropdownExpandableMenuItemProps extends AccordionItemProps {
+  /**
+   * If true, item will be disabled; it will appear dimmed and onClick event (or any other events) will not fire
+   * @default false
+   */
+  disabled?: boolean;
+  /**
+   * Leading icon for the menu item
+   */
+  icon?: React.ReactElement<IconProps>;
+  /**
+   * @internal
+   */
+  index?: number;
+  testId?: string;
+}
+
+export const DropdownExpandableMenuItem = React.forwardRef<
```

Comment:
If we are using `forwardRef`, we need to use it to avoid console warnings `(props, forwardedRef)`